### PR TITLE
Check the action name of RefreshStatistics against the lower case to prevent an error 500

### PR DESCRIPTION
**Pull request by ghost**
Check the action name of RefreshStatistics against the lower case to prevent an error 500

### 1. Why is this change necessary?
This change is necessary because at irregular intervals some browsers call the URL `widgets/index/refreshstatistic` instead of `widgets/index/refreshStatistic`. Without this change, this leads to an Error 500.

### 2. What does this change do, exactly?
All checks for the action name have been provided with a strtolower and it is always checked against the lower case.

### 3. Describe each step to reproduce the issue or behaviour.
call `widgets/index/refreshstatistic` instead of `widgets/index/refreshStatistic`

### 5. Which documentation changes (if any) need to be made because of this PR?
nothing

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: engine/Shopware/Components/HttpCache/AppCache.php [hunk 1]
```diff
@@ -99,7 +99,9 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
             return $this->pass($request, $catch);
         }
 
-        if (strpos($request->getPathInfo(), '/widgets/index/refreshStatistic') === 0) {
```
Comment (shyim) [inline engine/Shopware/Components/HttpCache/AppCache.php:105]: stripos?

Comment (viosys) [reply engine/Shopware/Components/HttpCache/AppCache.php:105]: thanks

Comment (pascalheidmann) [reply engine/Shopware/Components/HttpCache/AppCache.php:105]: As `stripos` is case insensitive you can still use `'/widgets/index/refreshStatistic'` as comparison and there should be no need for that `@noinspection` annotation.

(same goes for other instances)

**Review by pascalheidmann — APPROVED** [review]

**Comment by shyim** [comment]
Thanks for your Contribution! :blue_heart:

**Pull request closed without merging**